File: tests/fixtures/func_sample.c
// module-level constant - excluded with functions-only
const char *MODULE_STR = "module_string";
int MODULE_NUM = 200;

void myFunc() {
    const char *x = "func_string";
    int n = 77;
}

void anotherFunc(const char *param) {
    const char *msg = "another_func";
}

File: tests/fixtures/func_sample.rs
// module-level constant - excluded with functions-only
static MODULE_STR: &str = "module_string";
static MODULE_NUM: i32 = 200;

fn my_func() -> &'static str {
    let x = "func_string";
    let n: i32 = 77;
    x
}

fn another_func(param: &str) {
    let msg = "another_func";
    println!("{}", msg);
}
